src/SIM/SIMoutput.C

```diff
@@ -216,8 +216,12 @@ void SIMoutput::preprocessResPtGroup (std::string& ptFile, ResPointVec& points)
     else if ((p->inod = pch->evalPoint(p->u,p->u,p->X)) < 0)
       p = points.erase(p);
     else
+      (p++)->npar = pch->getNoParamDim();
+  }
+
+  if (points.size() < 5 || msgLevel > 1)
+    for (ResPointVec::const_iterator p = points.begin(); p != points.end(); ++p)
```

Comment:
range based for while at it ? unless there is some iterator usage hiding in lines not shown..

Reply:
Not really, except for then have to change all `->` into `.` Can do that of course